<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameInfoWidget</class>
 <widget class="QWidget" name="GameInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QSpinBox" name="heartContainersSpinBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="rupeesLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rupees:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="rupeesSpinBox"/>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="playerNameLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="heartContainersLbl">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Heart Containers:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="currentHealthLbl">
     <property name="text">
      <string>Current Health:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="currentHealthSpinBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="playerNameWidget">
     <property name="text">
      <string>Player Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="createGameChkBox">
     <property name="text">
      <string>Create Game</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>createGameChkBox</tabstop>
  <tabstop>playerNameLineEdit</tabstop>
  <tabstop>rupeesSpinBox</tabstop>
  <tabstop>heartContainersSpinBox</tabstop>
  <tabstop>currentHealthSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
